dashboard/test/controllers/plc_enrollment_evaluations_controller_test.rb

```diff
@@ -0,0 +1,100 @@
+    assert_equal 2, questions.second.plc_evaluation_answer.count, 'The second question should have two answers'
+    assert_equal 3, questions.third.plc_evaluation_answer.count, 'The third question should have three answers'
+    assert_equal 2, questions.fourth.plc_evaluation_answer.count, 'The fourth question should have two answers'
+    assert_equal 2, questions.fifth.plc_evaluation_answer.count, 'The fifth question should have two answers'
+  end
+
+  test "submit evaluation enrolls user in appropriate modules" do
+    #This can get simplified once I start using the hidden field in user submission
+
+    #Sir Lancelot says that his name is Lancelot, his quest is to seek the grail, and that his favorite color is blue.
+    #He should be enrolled only in "Answering questions honestly"
+    do_expected_answers_yield_expected_module_enrollments(
+        [@answer1_1.professional_learning_task_id, @answer2_1.professional_learning_task_id, @answer3_1.professional_learning_task_id].to_s, [@module3])
+
+    #Sir Robin says that his name is Robin, his quest is to seek the grail, and that he doesn't know the capital of Assyria
+    #He should be enrolled in "Answering questions honestly" and "Admitting Ignorance"
+    do_expected_answers_yield_expected_module_enrollments(
+        [@answer1_2.professional_learning_task_id, @answer2_1.professional_learning_task_id, @answer4_1.professional_learning_task_id].to_s, [@module3, @module4])
+
+    #Sir Galahad says that his name is Galahad, his quest is to seek the grail, and that his favorite color is blue - no, yellow
+    #He should be enrolled in "Answering questions honestly" and "Getting thrown off cliffs"
+    do_expected_answers_yield_expected_module_enrollments(
+        [@answer1_3.professional_learning_task_id, @answer2_1.professional_learning_task_id, @answer3_2.professional_learning_task_id].to_s, [@module3, @module1])
+
+    #King Arthur says that his name is Arthur, his quest is to seek the grail, and needs clarification on swallow speed
+    #He should be enrolled in "Answering questions honestly" and "Advanced Ornitholoy"
+    do_expected_answers_yield_expected_module_enrollments(
+        [@answer1_4.professional_learning_task_id, @answer2_1.professional_learning_task_id, @answer5_1.professional_learning_task_id].to_s, [@module3, @module2])
+
+    #Sir Edgecase of Edgecaseville should be enrolled in no modules
+    do_expected_answers_yield_expected_module_enrollments(
+        [@answer1_5.professional_learning_task_id, @answer3_3.professional_learning_task_id, @answer5_2.professional_learning_task_id].to_s, [])
+  end
+
+  private
+  def do_expected_answers_yield_expected_module_enrollments(answers, expected_module_enrollments)
+    post :submit_evaluation, enrollment_id: @plc_enrollment.id, answerTaskList: answers
+    assert_equal expected_module_enrollments.collect {|m| m.id}, @plc_enrollment.user_enrollment_module_assignment.all.collect {|m| m.professional_learning_module_id}
```

Comment:
style guide prefers map over collect. you can also use `&:`, oh, and .all is not necessary if you use Enumerable methods like map because it will know that you want to resolve the query:

+    assert_equal expected_module_enrollments.map(&:id), @plc_enrollment.user_enrollment_module_assignment.map(&:id)